usecases/backup/scheduler.go

```diff
@@ -128,6 +124,10 @@ func (s *Scheduler) Backup(ctx context.Context, pr *models.Principal, req *Backu
 		return nil, backup.NewErrUnprocessable(err)
 	}
 
+	if err := s.authorizer.Authorize(pr, authorization.CREATE, authorization.Backups(classes...)...); err != nil {
+		return nil, err
+	}
+
 	if err := s.authorizer.Authorize(pr, authorization.READ, authorization.Collections(classes...)...); err != nil {
 		return nil, err
 	}
```

Comment:
**Issue (blocking):** Now that we have backups filtered by collection, we should remove these permissions. Otherwise, a role that can backup needs to have `read_schema` and `read_data` on the whole collection allowing them to perform many other operations, e.g. GQL querying, etc.

Reply:
good catch !